### ISSUE TITLE
[Bug]: RuntimeError when do onnx export

### ISSUE THREAD

**Issue by OleksandrKorovii** [flairNLP/flair#3072]
### Describe the bug

I try reproduce Tutorial 13 and get next error RuntimeError

### To Reproduce

```python
from flair.models import SequenceTagger
from flair.embeddings import TransformerWordEmbeddings, TransformerDocumentEmbeddings

model = SequenceTagger.load("ner_model.pt")
assert isinstance(model.embeddings, (TransformerWordEmbeddings, TransformerDocumentEmbeddings))

from flair.datasets import CONLL_03
sentences = list(CONLL_03().test)[:5]

model.embeddings = model.embeddings.export_onnx("Ner-Model-Embeddings.onnx", sentences, providers=["CUDAExecutionProvider", "CPUExecutionProvider"])
```

### Expected behaivor

The embeddings convert into onnx format

### Logs and Stack traces

```stacktrace
File ~/ELL/flair-venv/lib/python3.10/site-packages/flair/embeddings/transformer.py:1279, in TransformerEmbeddings.forward(self, input_ids, sub_token_lengths, token_lengths, attention_mask, overflow_to_sample_mapping, word_ids, langs, bbox, pixel_values)
   1271     all_token_embeddings[:, :, sentence_hidden_states.shape[2] :] = fill_masked_elements(
   1272         all_token_embeddings[:, :, sentence_hidden_states.shape[2] :],
   1273         sentence_hidden_states,
   (...)
   1276         token_lengths,
   1277     )
   1278 elif self.subtoken_pooling == "mean":
-> 1279     all_token_embeddings = fill_mean_token_embeddings(
   1280         all_token_embeddings, sentence_hidden_states, word_ids, token_lengths
   1281     )
   1282 else:
   1283     raise ValueError(f"subtoken pooling method: `{self.subtoken_pooling}` is not implemented")

File ~/ELL/flair-venv/lib/python3.10/site-packages/torch/jit/_trace.py:1139, in _script_if_tracing.<locals>.wrapper(*args, **kwargs)
   1136     return fn(*args, **kwargs)
   1138 compiled_fn = script(wrapper.__original_fn)  # type: ignore[attr-defined]
-> 1139 return compiled_fn(*args, **kwargs)

RuntimeError: fill_mean_token_embeddings() expected at most 3 argument(s) but received 4 argument(s). Declaration: fill_mean_token_embeddings(Tensor all_token_embeddings, Tensor sentence_hidden_states, Tensor word_ids) -> Tensor

​
```

##### Flair
0.11.3 (installed from GitHub)
##### Pytorch
1.13.1+cu117
##### Transformers
4.25.1
#### GPU
True

**Comment by iliaNecrov** [comment]
Facing same issue in tagger predict method

**Comment by iliaNecrov** [comment]
Managed to fix the error in my case:

1. Downloaded `flair-0.11.3-py3-none-any.whl` from pypi
2. In /flair-0.11.3.dist-info/METADATA of the wheel file I deleted dependencies for `transformers` and `sentencepeice `and added this line: 
`Requires-Dist: transformers[sentencepiece] (>=4.0.0)`
(We need this to avoid `sentencepeice` error while installing `flair`)
3. Then locally installed corrected .whl file via `pip`

@OleksandrKorovii may be useful for you
